_overviews/core/collections-migration-213.md

```diff
@@ -22,14 +23,38 @@ The most important changes in the Scala 2.13 collections library are:
   - Deprecated collections were removed (`MutableList`, `immutable.Stack`, others)
   - Parallel collections are now in a separate hierarchy in a [separate module](https://github.com/scala/scala-parallel-collections).
   - The `scala.jdk.StreamConverters` object provides extension methods to create (sequential or parallel) Java 8 streams for Scala collections.
-  - `scala.Seq` is now an alias for `scala.collection.immutable.Seq` (no longer `scala.collection.Seq`). Note that this also changes the type of Scala varargs methods.
 
 ## Tools for migrating and cross-building
 
 The [scala-collection-compat](https://github.com/scala/scala-collection-compat) is a library released for 2.11, 2.12 and 2.13 that provides some of the new APIs from Scala 2.13 for the older versions. This simplifies cross-building projects.
 
 The module also provides [migratrion rules](https://github.com/scala/scala-collection-compat#migration-tool) for [scalafix](https://scalacenter.github.io/scalafix/docs/users/installation.html) that can update a project's source code to work with the 2.13 collections library.
 
+## scala.Seq migration
+
+In Scala 2.13 `scala.Seq[+A]` is an alias for `scala.collection.immutable.Seq[A]`, instead of `scala.collection.Seq[A]`. This change requires some planning depending on how your code is going to be used.
+
+If you're making an application, and simply migrating a Scala 2.12 code base to 2.13, it might be ok to keep using `scala.Seq` in your code.
+
+If you're making a library intended to be used by other programmers, then using `scala.Seq` or varargs is going to be a breaking change in the API semantics. For example, if there was a function `def orderFood(order: Seq[Order]): Seq[Food]`, previously the library user would have been able to pass in an array of `Order`, but it won't work for 2.13.
+
+- if you cross build with Scala 2.12 and want to maintain the API semantics for 2.13 version of your library, or
+- if your library users frequently uses mutable collections such as `Array`
+
+you can import `scala.collection.Seq` ("CSeq") explicitly in your code.
```

Comment:
It might be worth mentioning that this is a potentially source-breaking change. If a user in 2.12 has:

```
// `Seq` is the default `scala.Seq`
val a: Seq[Food] = orderFood(myOrder)
```

Then that will no longer compile in 2.13

Reply:
Also, it might be a good strategy for APIs to only move to `immutable.Seq` in the result type, but keep accepting `collection.Seq` arguments, if the method implementation can work fine with mutable collections,.

Reply:
I've added this in 2b4de2d.